terraform/bastion/Jenkinsfile

```diff
@@ -0,0 +1,70 @@
+library("tdr-jenkinslib")
```

Comment:
I thought about it but because we're just auto approving and not doing the plan and everything and there are other parameters I thought it'd be easier just to have its own Jenkinsfile.